docs/kafka/metrics-monitoring/README.adoc

```diff
@@ -90,11 +90,11 @@ Broker metrics::
 Topic metrics::
 +
 --
-* `kafka_server_brokertopicmetrics_bytes_in_total`: Number of incoming bytes to topics in the instance
+* `kafka_topic:kafka_server_brokertopicmetrics_bytes_in_total:rate5m`: Rate of incoming bytes to a topic in the instance in the last five minutes
 
-* `kafka_server_brokertopicmetrics_bytes_out_total`: Number of outgoing bytes from topics in the instance
+* `kafka_topic:kafka_server_brokertopicmetrics_bytes_out_total:rate5m`: Rate of outgoing bytes from a topic in the instance in the last five minutes
 
-* `kafka_server_brokertopicmetrics_messages_in_total`: Number of messages per second received by one or more topics in the instance
+* `kafka_topic:kafka_server_brokertopicmetrics_messages_in_total:rate5m`: Rate of messages per second received by a topic in the instance in the last five minutes
 
```

Comment:
After recent discussions it's been agreed on that we won't be removing the old metrics. Let's add these new ones to the end of the list then

Reply:
thanks @JameelB - I have updated my PR